models/junos/l3_interfaces/overridden_example_01.txt

```diff
@@ -0,0 +1,59 @@
+# Using overridden
+
+# Before state
+# ------------
+#
+# admin# show interfaces
+# ge-0/0/1 {
+#     description "L3 interface";
+#     unit 0 {
+#         family inet {
+#             address 10.200.16.10/24;
+#         }
+#     }
+# }
+# ge-0/0/2 {
+#     description "L3 interface with dhcp";
+#     unit 0 {
+#         family inet {
+#             dhcp;
+#         }
+#     }
+# }
+
+- name: Override provided configuration with device configuration
+  junos_l3_interfaces:
+    config:
+      - name: ge-0/0/1
+        ipv4:
+           - address: 192.168.1.10/24
+        ipv6:
+           - address: 8d8d:8d01::1/64
+      - name: ge-0/0/2
+        ipv6:
+           - address: 2001:db8:3000::/64
+    state: overridden
```

Comment:
In the case of overridden if other interfaces have l3 related configuration apart from ge-0/0/1 and ge-0/0/2 it should be deleted in this example. Probably one more interface say ge-0/0/3 can be added in before config and show the l3 config is deleted from ge-0/0/3 in the after state.